dotcom-rendering/src/components/FootballMatchList.tsx

```diff
@@ -367,7 +367,18 @@ export const FootballMatchList = ({
 					{day.competitions.map((competition) => (
 						<Fragment key={competition.competitionId}>
 							<CompetitionName>
-								{competition.name}
+								<a
+									href={`https://www.theguardian.com/football/${competition.tag}`}
```

Comment:
The host is available within the config object that comes from the Frontend data model. It's the same value that gets set on `window.guardian.config.page.host`

This would allow us to set the correct link for the CODE environment

If the component is deeply nested and its a pain to prop drill you could also consider adding it to [ConfigContext](https://github.com/guardian/dotcom-rendering/blob/a84ea2e85e9ac2c8b6ccf0f2704280795ce70b44/dotcom-rendering/src/components/ConfigContext.tsx) and then use the `useConfig` hook from the same module

Reply:
Could we just use the relative url?

Reply:
(this would obviously break in storybook)

Reply:
Yes relative would work too. I think using the value from config is a little more explicit but it is a very minor point.